python/lsst/meas/base/forcedMeasurement.py

```diff
@@ -298,16 +302,20 @@ def run(self, exposure, refCat, refWcs, idFactory=None, exposureId=None):
             # TODO: skip this loop if there are no plugins configured for single-object mode
             for refChildRecord, measChildRecord in zip(refChildCat, measChildCat):
                 noiseReplacer.insertSource(refChildRecord.getId())
-                self.callMeasure(measChildRecord, exposure, refChildRecord, refWcs)
+                self.callMeasure(measChildRecord, exposure, refChildRecord, refWcs,
+                    beginOrder=beginOrder, endOrder=endOrder)
                 noiseReplacer.removeSource(refChildRecord.getId())
 
             # then process the parent record
             noiseReplacer.insertSource(refParentRecord.getId())
-            self.callMeasure(measParentRecord, exposure, refParentRecord, refWcs)
+            self.callMeasure(measParentRecord, exposure, refParentRecord, refWcs,
+                    beginOrder=beginOrder, endOrder=endOrder)
             self.callMeasureN(measParentCat[parentIdx:parentIdx+1], exposure,
-                              refParentCat[parentIdx:parentIdx+1])
+                              refParentCat[parentIdx:parentIdx+1],
+                    beginOrder=beginOrder, endOrder=endOrder)
```

Comment:
I'm okay with either indenting to the open-parentheses or indenting by a fixed amount, but it should be made consistent here.